### ISSUE TITLE
Small cell carcinoma of the ovary, hypercalcaemic type (SCCOHT).

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7566]
**Preferred term label:**
Small cell carcinoma of the ovary, hypercalcaemic type (SCCOHT).

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
unsure what the motivation was, closing for now
